cmd/talosctl/cmd/talos/config.go

```diff
@@ -175,8 +175,40 @@ func openConfigAndContext(context string) (*clientconfig.Config, error) {
 	return c, nil
 }
 
+// configMergeCmd represents the config merge command.
+var configMergeCmd = &cobra.Command{
+	Use:    "merge <from>",
+	Short:  "Merge in additional contexts from another Talos config",
+	Long:   ``,
+	Hidden: false,
+	Args:   cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		from := args[0]
+		c, err := clientconfig.Open(Talosconfig)
```

Comment:
I don't remember, but does it handle the case when `~/.talos/config` is missing?

Reply:
yep, it handles that case.